components/availability/src/index.html

```diff
@@ -69,61 +69,61 @@
         });
 
         document.querySelectorAll('input[name="slot-size"]').forEach((elem) => {
-          elem.addEventListener("change", function (event) {
+          elem.addEventListener("input", function (event) {
             component.slot_size = event.target.value;
           });
         });
 
         document
-          .querySelectorAll('input[name="click-action"]')
+          .querySelectorAll('input[name="show-ticks"]')
           .forEach((elem) => {
             elem.addEventListener("change", function (event) {
-              component.click_action = event.target.value;
+              component.show_ticks = JSON.parse(event.target.value);
```

Comment:
Created a new CH story to make this consistent with other components:
https://app.clubhouse.io/nylas/story/65337/availability-component-update-exported-props-to-use-getpropertyvalue-to-make-it-consistent-with-other-components

For now though, I say leave this as is!